tests/unit/test_repos_repo.py

```diff
@@ -144,7 +144,21 @@ def test_create_key(self):
         """Verify the request to create a key."""
         data = {
             'title': 'octocat@octomac',
-            'key': 'ssh-rsa AAA'
+            'key': 'ssh-rsa AAA',
+            'read_only': 'false'
+        }
+        self.instance.create_key(**data)
```

Comment:
Please expand this. Your tests above pass `read_only=True` or `read_only=False` but instead are passing strings here. Instead of splatting data, please just pass the arguments directly and do the same below.